### ISSUE TITLE
Glob file issue with latest build

### ISSUE THREAD

**Issue by wyardley** [koalaman/shellcheck#2546]
#### For bugs
- Rule Id (if any, e.g. SC1000): n/a
- My shellcheck version (`shellcheck --version` or "online"):  `v0.8.0-53-g4a27c9a` (note: this issue just started showing up with the`latest` tag on Docker)
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
shellcheck **/*.sh
```

#### Here's what shellcheck currently says:
```
shellcheck: Prelude.foldl1: empty list
```

#### Here's what I wanted or expected to see:
[no output if no errors]

**Comment by koalaman** [comment]
Fixed in a30ac40. Thanks!

**Comment by wyardley** [comment]
Thanks for the quick fix!
